File: a095(conspiracy)/C++/conspiracy.cpp
#include <iostream>

int main(void) {
  int prisoners, hats;
  while (std::cin >> prisoners >> hats) {
    if (prisoners > hats) std::cout << hats + 1 << '\n';
    else if (prisoners == hats) std::cout << hats << '\n';
    else break; 
  }

  return 0;
}

File: a095(conspiracy)/Python/conspiracy.py
while True:
    try:
        prisoners, hats = map(int, input().split())
        if prisoners > hats:
            print(hats + 1)
        elif prisoners == hats:
            print(hats)
        else:
            raise ValueError('There can\'t be more hats than prisoners!')
    except:
        break
